src/logs/logs.proto

```diff
@@ -72,6 +74,11 @@ message PipelineJobLogQuery {
   string job = 2;
 }
 
+message JobDatumLogQuery {
+  string job = 1;
```

Comment:
Should this be a PPS Job message (i.e., pipeline + job ID) or a job string?

Reply:
Notably, the existing pipeline query doesn’t use a PPS pipeline message, but is structured into project name and pipeline name, while the existing job query is structured into a pipeline (= project name and pipeline name) and job ID. Maybe JobDatumLogQuery should similarly use a PipelineJobLogQuery?

Reply:
Ah, makes sense because we can uniquely identify with the datum & job IDs together — no need for the pipeline-job ID.